demo/src/main/resources/db/changelog/BB-02_Create_test_tables.sql

```diff
@@ -1,21 +1,62 @@
 -- liquibase formatted sql
 -- changeset OleksiiSkachkov:BB-02_Create_test_tables
 
-create table users (
+create table if not exists users (
```

Comment:
could you please also update the same file in demo module?

Reply:
done